audit/result.go

```diff
@@ -28,7 +29,9 @@ type Result struct {
 // See also Unmarshal.
 func (r *Result) Marshal() []byte {
 	if !r.versionEncoded {
-		r.v2.SetVersion(version.Current().ToV2())
+		var verV2 refs.Version
+		version.Current().WriteToV2(&verV2)
```

Comment:
how about adding an empty line everywhere after the reading/writing with new `WriteToV2` `ReadFromV2`?

Reply:
I don't know how this code will look like after refactor of all packages, so I decided to ignore style preferences and make it compilable.